src/site/components/merger-crosslinks.html

```diff
@@ -26,7 +26,7 @@
   {% for group in crosslinks %}
 
     {% if group.heading != empty %}
-      <h3 class="va-nav-linkslist-heading">{{ group.heading }}</h3>
+      <h3 class="va-nav-linkslist-heading" tabindex="-1">{{ group.heading }}</h3>
```

Comment:
@1Copenut Does this look okay to you?

Reply:
added per his request in https://github.com/department-of-veterans-affairs/va.gov-team/issues/3580

Reply:
Awesome. Thanks for the cross-reference!